src/test/java/com/uid2/admin/salt/SaltRotationTest.java

```diff
@@ -483,4 +489,241 @@ void testRotateSaltsZeroWorksWhenThereIsFutureSaltFile() throws Exception {
         assertThat(result.getSnapshot().getEffective()).isEqualTo(targetDate().asInstant());
         assertThat(result.getSnapshot().getExpires()).isEqualTo(daysLater(7).asInstant());
     }
+
+    @Test
+    void testRotateFromSaltToKey() throws Exception {
```

Comment:
Can we consolidate testRotateFromSaltToKey and testKeyRotationInitialKeyIdPopulation?